### ISSUE TITLE
"Sign in With Apple" feature apparently causing IE 11 error

### ISSUE THREAD

**Issue by sgomes** [Automattic/wp-calypso#38662]
#### Steps to reproduce
1. Open IE 11
2. Open the JS console
3. Open the following URL: https://wordpress.com/log-in

#### What I expected
No errors in console output

#### What happened instead
The following error is present in the console output:
```
SCRIPT445: Object doesn't support this action
appleid.auth.js (26,13285)
```

#### Browser / OS version
Internet Explorer 11 running on Windows 10

#### Screenshot / Video
![image](https://user-images.githubusercontent.com/409615/71834395-0896e580-30a7-11ea-9c3b-7169f64fc78e.png)

#### Context / Source
The error appears to be coming from `appleid.auth.js`.

**Comment by kriskarkoski** [comment]
Tested in Browserstack and I cannot reproduce this issue. @sgomes I'm going to close, but please reopen if you can reproduce.

**Comment by sgomes** [comment]
Thank you, @kriskarkoski! I'm guessing fixes elsewhere (like perhaps #40075) have fixed this.

**Comment by willbokulet** [comment]
I'm encountering this.. what might be the possible issue and if there is a solution?

![image](https://user-images.githubusercontent.com/37994506/89273919-d41be580-d605-11ea-97ea-2f1c1145cb63.png)
